src/collections/posts/getting-started-with-mapping.md

```diff
@@ -3,7 +3,7 @@ title: Getting Started with Mapping Levels
 section: articles
 category: guide
 publish: 2024-01-31T00:00:00.000Z
-homepageText: Getting Started with Mapping Levels
+homepageText: There are more than 100,000 maps on BeatSaver but you still may not be able to find the song you want. Anyone can learn to make their own maps with the resources available on this page!
```

Comment:
The latest key points towards 243734 maps, albeit deleted and AI maps are included.